### PR TITLE
New Omnath and support

### PR DESCRIPTION
**Pull request by tool4ever**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/omnath_locus_of_all.txt [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Name:Omnath, Locus of All
+ManaCost:W U P R G
+Types:Legendary Creature Phyrexian Elemental
+PT:4/4
+R:Event$ LoseMana | ValidPlayer$ You | ReplacementResult$ Replaced | ReplaceWith$ ConvertMana | ActiveZones$ Battlefield | Description$ If you would lose unspent mana, that mana becomes black instead.
+SVar:ConvertMana:DB$ ReplaceMana | ReplaceType$ B
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/upcoming/omnath_locus_of_all.txt:6]: Problem: Omnath uses ColorWord, that means it should be affected by CardTextChanges, that's why it needs to be "Black"

File: forge-game/src/main/java/forge/game/ability/effects/ReplaceManaEffect.java [hunk 2]
```diff
@@ -39,7 +39,7 @@ public void resolve(SpellAbility sa) {
             }
         } else if (sa.hasParam("ReplaceType")) {
             // replace color and colorless
-            String color = sa.getParam("ReplaceType");
+            String color = MagicColor.toShortString(sa.getParam("ReplaceType"));
```
Comment (Hanmac) [inline forge-game/src/main/java/forge/game/ability/effects/ReplaceManaEffect.java:42]: it does collide with the "Any" below

File: forge-gui/res/cardsfolder/upcoming/omnath_locus_of_all.txt [hunk 3]
```diff
@@ -0,0 +1,14 @@
+Name:Omnath, Locus of All
+ManaCost:W U P R G
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/upcoming/omnath_locus_of_all.txt:2]: the mana cost should be PB  for Phyrexian Black?

File: forge-game/src/main/java/forge/game/ability/effects/ReplaceManaEffect.java (hunk 2 above)
Comment (tool4ever) [reply forge-game/src/main/java/forge/game/ability/effects/ReplaceManaEffect.java:42]: 🤦‍♂️

**Comment by Northmoc** [comment]
Nice work – does this then ask you to order the replacements if you have this Omnath and one of the "lost becomes C" things?

**Comment by tool4ever** [comment]
yes

**Review by kevlahnota — APPROVED** [review]

**Pull request merged**